### ISSUE TITLE
IE9 - issue with the use of the ArrayBuffer object

### ISSUE THREAD

**Issue by sonnyk22** [maurizzzio/function-plot#32]
It seems that the Function-Plot library does not works in IE9 due to the use of the ArrayBuffer which is not supported until IE10.  Is there a possibility to replace the ArrayBuffer object with something like "polyfill" (or a better option) shown in below link:

http://stackoverflow.com/questions/16545286/internet-explorer-error-script5009-arraybuffer-is-undefined

**Comment by maurizzzio** [comment]
that's right, thanks I'll try to update the interval-arithmetic package in the weekend

**Comment by sonnyk22** [comment]
awesome!  thank you.

**Comment by maurizzzio** [comment]
Well unfortunately after applying the polyfill the `interval-arithmetic` library got painfully slow besides not getting 100% correct results, e.g. computing simple trigonometric functions

```javascript
trigonometric
      ✓ should compute the cosine function (599ms)
      ✓ should compute the sine function (507ms)
      ✓ should compute the tangent function (459ms)
```

Anyway I think it works, fixed in v1.9.3

**Comment by sonnyk22** [comment]
It works great in IE9 now.  Thank you for fixing this issue.
